tests/calico_testcases/example_policies/expected_output/testcase15_with_ingress_connectivity_map.txt

```diff
@@ -0,0 +1,56 @@
+final fw rules for query: connectivity_map, config: ip:
+src: 0.0.0.0/0 dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src: ::/0 dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [default,kube-system,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [default,kube-system,vendor-system] src_pods: [*] dst: ::/0 conn: All connections
+src_ns: [default,vendor-system] src_pods: [*] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [calico-node] conn: TCP {'dst_ports': '210', 'paths': '/abc(/(([\\--9A-Z_abce-z]|d([\\--9A-Z_a-df-z]|e([\\--9A-Z_a-eg-z]|f[\\-.0-9A-Z_a-z])))[\\--9A-Z_a-z]*|d(e(f/)?)?))?', 'hosts': 'first.bar.com'}
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [file-plugin-7bfb8b69bf] conn: TCP {'dst_ports': '80', 'paths': '/abc/def(/[\\--9A-Z_a-z]+)?', 'hosts': 'first.bar.com'}
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [kube-dns-amd64-d66bf76db] conn: TCP {'dst_ports': '213', 'paths': '/[\\--9A-Z_a-z]*'}
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [kube-fluentd] conn: TCP {'dst_ports': '80', 'paths': '/xyz/[\\--9A-Z_a-z]+', 'hosts': 'second.bar.com'}
+src_ns: [kube-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [storage-watcher-8494b4b8bb] conn: TCP {'dst_ports': '102', 'paths': '/xyz', 'hosts': 'second.bar.com'}
+src_ns: [kube-system] src_pods: [calico-kube-controllers-7694668c77] dst_ns: [default,kube-system,vendor-system] dst_pods: [*] conn: All connections
```

Comment:
I am not sure that I understand this expected output... I'd expect Ingress to affect only on the connectivity from the ingress-controller pod..

Reply:
I think this is another way to define the limited connectivity from the ingress controller pods: limit connectivity from all pods in kube-system namespace, and add 'All connections' from all but ingress controller pods in this namespace.

Reply:
Right, ok